pkg/kubectl/cmd/cmd.go

```diff
@@ -219,69 +219,64 @@ func NewKubectlCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 
 	groups := templates.CommandGroups{
 		{
-			Message: "Basic Commands (Beginner):",
+			Message: "Application Management (Using Command Args / Flags)",
 			Commands: []*cobra.Command{
-				NewCmdCreate(f, out),
+				NewCmdAnnotate(f, out),
+				NewCmdAutoscale(f, out),
 				NewCmdExposeService(f, out),
+				NewCmdLabel(f, out),
+				rollout.NewCmdRollout(f, out),
 				NewCmdRun(f, in, out, err),
+				NewCmdScale(f, out),
 				set.NewCmdSet(f, out, err),
 			},
 		},
 		{
-			Message: "Basic Commands (Intermediate):",
+			Message: "Application Management (Either Yaml Or Command Args)",
 			Commands: []*cobra.Command{
-				NewCmdGet(f, out, err),
-				NewCmdExplain(f, out, err),
-				NewCmdEdit(f, out, err),
+				NewCmdCreate(f, out),
 				NewCmdDelete(f, out),
-			},
-		},
-		{
-			Message: "Deploy Commands:",
-			Commands: []*cobra.Command{
-				rollout.NewCmdRollout(f, out),
+				NewCmdGet(f, out, err),
 				NewCmdRollingUpdate(f, out),
-				NewCmdScale(f, out),
-				NewCmdAutoscale(f, out),
 			},
 		},
 		{
-			Message: "Cluster Management Commands:",
+			Message: "Application Management (Using Yaml Config)",
```

Comment:
I don't know that most new users have any idea what YAML is, so this has the potential for confusion.  "Application Management (via config files)" might be better.